app/scripts/controllers/metamask-notifications/constants/notification-schema.ts

```diff
@@ -0,0 +1,163 @@
+export enum TRIGGER_TYPES {
+  FEATURES_ANNOUNCEMENT = 'features_announcement',
+  METAMASK_SWAP_COMPLETED = 'metamask_swap_completed',
+  ERC20_SENT = 'erc20_sent',
+  ERC20_RECEIVED = 'erc20_received',
+  ETH_SENT = 'eth_sent',
+  ETH_RECEIVED = 'eth_received',
+  ROCKETPOOL_STAKE_COMPLETED = 'rocketpool_stake_completed',
+  ROCKETPOOL_UNSTAKE_COMPLETED = 'rocketpool_unstake_completed',
+  LIDO_STAKE_COMPLETED = 'lido_stake_completed',
+  LIDO_WITHDRAWAL_REQUESTED = 'lido_withdrawal_requested',
+  LIDO_WITHDRAWAL_COMPLETED = 'lido_withdrawal_completed',
+  LIDO_STAKE_READY_TO_BE_WITHDRAWN = 'lido_stake_ready_to_be_withdrawn',
+  ERC721_SENT = 'erc721_sent',
+  ERC721_RECEIVED = 'erc721_received',
+  ERC1155_SENT = 'erc1155_sent',
+  ERC1155_RECEIVED = 'erc1155_received',
+}
+
+export enum TRIGGER_TYPES_GROUPS {
+  RECEIVED = 'received',
+  SENT = 'sent',
+  DEFI = 'defi',
+}
+
+export const NOTIFICATION_CHAINS = {
+  ETHEREUM: '1',
+  OPTIMISM: '10',
+  BSC: '56',
+  POLYGON: '137',
+  ARBITRUM: '42161',
+  AVALANCHE: '43114',
+  LINEA: '59144',
+};
```

Comment:
We are keeping a local copy of the chains here as we need this controller to be loosely coupled with the extension (it will need to be reused with other platforms).

If there is a nice reusable list of chains from a shared package that would be perfect!

Reply:
We do not have a reusable list of chains in a shared package at this time.

Reply:
no worries that is fine!